server/command.go

```diff
@@ -229,5 +229,7 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 		return &model.CommandResponse{}, nil
 	}
 
+	p.postCommandResponse(args, fmt.Sprintf("Unknown action %v", action))
```

Comment:
Future improvement may be to return the output of `/github help`, so the user can immediately see what commands are available

Reply:
0/5 that just printing "Unknown action" is enough here. Your call.

Reply:
Yes, my note was intended as a future improvement, this PR alone is an improvement (it's better to return a message than no message at all)